### DISCUSSION TITLE
Sync only one of two libraries

### DISCUSSION THREAD

**Discussion by iNfRaS0NiC** [Taxel/PlexTraktSync#1002] [Q&A] [answered]
Hello guys, I have two Movie libraries, one for me and my wife and one for my son. He is watching repeatedly some movies and I don't want to mess with my Trakt watch history.
Is there a way to choose not to sync one of the two Movies libraries?
Thank you for this amazing work and keep it up.

**Comment by simonc56** [comment, marked as answer]
You have `excluded-libraries` in config.yml
